kuma/settings/common.py

```diff
@@ -45,7 +45,9 @@ def __call__(self, value):
 STAGING_DOMAIN = 'developer.allizom.org'
 STAGING_URL = PROTOCOL + STAGING_DOMAIN
 
-INTERACTIVE_EXAMPLES_BASE = 'https://interactive-examples.mdn.mozilla.net'
+INTERACTIVE_EXAMPLES_BASE = config(
+    'interactive-examples.mdn.mozilla.net',
```

Comment:
Should this be ``https://interactive-examples.mdn.mozilla.net``?

Reply:
Crud, indeed. Copy/paste fail. Let me update it.

Reply:
Fixed up.